src/components/Result/ResultTable.vue

```diff
@@ -1,22 +1,22 @@
 <template>
-	<div class="px-6 md:px-0 container mx-auto my-10">
+	<div class="md:container mx-1 md:mx-auto my-2 md:my-10">
 		<div>
 			<div
-				class="flex justify-start mb-2 text-gray-500 text-md uppercase"
+				class="flex justify-start mb-2 md:mx-6 px-2 text-gray-500 text-md uppercase"
 			>
-				<span class="w-6/12 pl-12"> Súbor </span>
-				<span class="w-3/12 text-center"> Percentuálna zhoda </span>
+				<span class="w-5/12 md:w-6/12 pl-1 md:pl-4 text-xs md:text-lg"> Súbor </span>
```

Comment:
You know you can add the `text` classes to parent element, you don't have to add it to the every single element.